packages/cursorless-org-docs/src/docs/user/release-notes/1.0.md

```diff
@@ -0,0 +1,19 @@
+---
+version: "1.0"
+release_date: "2025-03-04"
+---
+
+# Release Notes for 1.0
+
+Cursorless if finally ready for its 1.0 release!
+
+Over the last year the Cursorless team has focused primarily on the user experience and migrating legacy language scopes. We are now at a point where we feel Cursorless is ready for its first official release.
```

Comment:
I went with the terminology `official release`. I didn't want to use `stable` because that implies that Cursorless was unstable before.